support/approved-access.adoc

```diff
@@ -0,0 +1,20 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="approved-access"]
+= Approved Access
+include::_attributes/common-attributes.adoc[]
+ifdef::openshift-dedicated[]
+include::_attributes/attributes-openshift-dedicated.adoc[]
+endif::[]
+:context: approved-access
+
+toc::[]
+
+Red{nbsp}Hat Site Reliability Engineering (SRE) managing and proactively supporting ROSA Clusters will typically not require access to customer data as part of the normal operations. In the unlikely event should you need access to customer data, the *Approved Access* functionality provides an interface for customers to review and *approve* or *deny* access requests.
+
+Access requests to customer data on ROSA clusters and the corresponding cloud accounts can be created by Red{nbsp}Hat SRE either in response to a customer-initiated support ticket or in response to alerts received by a Red{nbsp}Hat SRE, as part of the standard incident response process.
+
```

Comment:
@fmcdonal I think this section would benefit from a couple of additional points:

- We should inform customers that Approved Access is an opt-in service
- I think a section on "Why choose to Enable Approved Access?" would be useful to help customers make a decision e.g
   -  You have regulatory requirements that mean you must audit all access to systems potentially containing customer data.
   - ...
- Does it make more sense to inform the customer here about what happens if they do not respond to Access Requests?
  - Inform the customer that turning on Approved Access means that they need to respond in a timely manner otherwise there is a risk to the SLA of ROSA
  - Inform the customer what will happen if they deny Access to Red Hat SRE

Reply:
@robpblake Do you feel this is not covered already in the "https://77370--ocpdocs-pr.netlify.app/openshift-rosa/latest/support/approved-access.html#support-reviewing-an-access-request-from-an-email-notification_approved-access" where we have notes about the "Deny" and "SLA"

Reply:
@fmcdonal Yes it is covered in that section. My suggestion is we move it to the introductory section about Approved Access to allow customers to make an informed decision about whether or not to turn Approved Access on.

However customers approve/deny Access Requests, they should be made aware of the implications of being slow to respond as part of their consideration to turn it on.

Reply:
applied